scripts/mp-2023/plot_multicore_performance_over_cores.py

```diff
@@ -12,7 +12,7 @@
 import json
 
 #configure directory with benchmark results
-benchmark_results_dir = "./scripts/results_variable_core_new_master"
+benchmark_results_dir = "./scripts/benchmark_hyrise_master_variable_core_23_01_04"
```

Comment:
could be made more general

Reply:
always needs to be configured either way